packages/frontend/core/src/commands/affine-creation.tsx

```diff
@@ -31,7 +32,7 @@ export function registerAffineCreationCommands({
       run() {
         track.$.cmdk.creation.createDoc({ mode: 'page' });
 
-        pageHelper.createPage('page');
+        pageHelper.createPage('page' as DocMode);
```

Comment:
Better to use `DocMode.Page` instead of `as DocMode` here?

Reply:
It breaks the unit test somehow. So I have to use type import here.